hazelcast/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -267,19 +295,38 @@ public void run() {
         } else if ("silent".equals(first)) {
             silent = Boolean.parseBoolean(args[1]);
         } else if ("shutdown".equals(first)) {
-            hazelcast.getLifecycleService().shutdown();
+            client.getLifecycleService().shutdown();
         } else if ("echo".equals(first)) {
             echo = Boolean.parseBoolean(args[1]);
             println("echo: " + echo);
+        } else if (equalsIgnoreCase(first, "clear")) {
+            lineReader.getTerminal().puts(InfoCmp.Capability.clear_screen);
+        } else if (equalsIgnoreCase(first, "history")) {
+            History hist = lineReader.getHistory();
+            ListIterator<History.Entry> iterator = hist.iterator();
+            while (iterator.hasNext()) {
+                History.Entry entry = iterator.next();
+                if (iterator.hasNext()) {
+                    String entryLine = new StringBuilder()
```

Comment:
I see that we use something like 
```java
new AttributedStringBuilder().style(AttributedStyle.BOLD.foreground(PRIMARY_COLOR))....
```
for the history command in `HazelcastCommandLine`. Would it be better if we use the same approach here?